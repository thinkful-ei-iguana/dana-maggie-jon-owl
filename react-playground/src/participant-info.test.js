import React from 'react';
import ReactDOM from 'react-dom';
import IndividualParticipant from './participant-info'
import renderer from 'react-test-renderer';

describe('<IndividualParticipant Component />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(<IndividualParticipant name="Peter" avatar="https://tf-assets-prod.s3.amazonaws.com/avatars/ea85f0a2-aaa2-4da6-936f-29dfe3bd78a6_MaggieMcClellan.png" inSession="true" onStage="false"/>, div);
        
        ReactDOM.unmountComponentAtNode(div);

    })

    it('renders the UI as expected', () => {

        const tree = renderer
            .create(<IndividualParticipant />)
            .toJSON();
        expect(tree).toMatchSnapshot();

    });

});