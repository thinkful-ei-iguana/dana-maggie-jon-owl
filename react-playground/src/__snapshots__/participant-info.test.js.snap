// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndividualParticipant Component /> renders the UI as expected 1`] = `
<div
  className="session-participant"
>
  <img
    className="user-avatar"
  />
  <div
    className="participant-info"
  >
    <h6
      className="participant-name"
    />
    <div
      className="participant-status participant-status__grey"
    >
      in session
    </div>
  </div>
</div>
`;

exports[`<Participant /> renders the UI as expected 1`] = `
<div
  className="session-participant"
>
  <img
    className="user-avatar"
  />
  <div
    className="participant-info"
  >
    <h6
      className="participant-name"
    />
    <div
      className="participant-status participant-status__grey"
    >
      in session
    </div>
  </div>
</div>
`;
